hirte/src/common/parse-util.c

```diff
@@ -40,3 +42,128 @@ bool parse_port(const char *in, uint16_t *ret) {
         *ret = (uint16_t) l;
         return true;
 }
+
+bool is_IP(char *str) {
```

Comment:
I think we can use [inet_pton](https://man7.org/linux/man-pages/man3/inet_pton.3.html) instead of a custom implementation.

Reply:
@engelmi Great suggestion!

IMHO, @mkemel is right and we shouldn't reinvent the wheel for anything in this project because of the mission critical environment in which it will run, and the FuSa focus of the overarching project.  We should bring in utility functions/macros from systemd anywhere possible.